README.md

```diff
@@ -41,7 +42,7 @@ available at [http://localhost:3000/](http://localhost:3000/).
 
 #### Unit Test
 
-The unit tests are powered by [chai](http://chaijs.com/) and [mocha](http://mochajs.org/).
+The unit tests are powered by [Jest](https://facebook.github.io/jest/), [Enzyme](http://airbnb.io/enzyme/) and [Sinon.JS](http://sinonjs.org/)
```

Comment:
Ok, question still stands. From what I understand now e2e tests use Mocha but unit tests use Jest. I'd like for us to use the same frameworks in both places.